### PR TITLE
Drop obsolete python

### LINKED ISSUE #11: Revitalize package
This repository is the base implementation for `z3c.relationfield` which is used in `plone.app.relationfield`.

There is #7 trying to fix the issued in Python 3.7+ but it is stalled for months.

As this repository seems to be an integral building block of Plone, I'd like to ask for a helping hand to get it fixed.
(I have no personal interest in this repository, the only thing I feel able to do is to provide infrastructure to run the tests.)

Cc-ing to a bunch of developers coming to my mind:

@mauritsvanrees @jensens @gforcada

Feels free to invite other developers which know this package or depend on its functionality.

### PR DESCRIPTION
**Pull request by ale-rt**
Closes #11

### CODE REVIEW

**Review by gforcada — APPROVED** [review]

File: .meta.toml [hunk 1]
```diff
@@ -33,12 +30,9 @@ testenv-deps = [
...
 [coverage]
-fail-under = 0
+fail-under = 91.66
 
 [github-actions]
-additional-config = [
-    "- [\"2.7\",   \"py27\"]",
-    ]
 additional-install = [
     "pip install -U \"tox < 4\"",
```
Comment (icemac) [inline .meta.toml]: This should also be removed. It was only necessary because tox 4.x no longer supports Python 2.

**Review by icemac — APPROVED** [review]
Mostly LGTM.

**Comment by icemac** [comment]
@ale-rt Please run again `meta/config` (using the `--branch` option to set the branch)  to get the changes you made in `.meta.toml` to `tests.yaml`.

**Review by icemac — APPROVED** [review]
LGTM.

**Pull request merged**